### PR TITLE
Feature/set up project

### PR DESCRIPTION
**Pull request by mohamedgamaleldin999999**
in this PR, I am trying to set up data model for my blog app following these requirements:

Create a new app. Give it a meaningful name.
Initialize your project with Git.
Make sure that your project has Postgres database set up.
Build your project schema.
Create and run the necessary migration files.
Table and column names should match the ERD diagram.
note: photo for users table should be a link to an image
Foreign keys should be included.
Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
All columns that are foreign keys should have a corresponding index.
Be sure to reference the from the sneak peek if you have any questions about table or column names or data relationships.

### CODE REVIEW

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @mohamedgamaleldin999999, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project without a proper README file. Please ensure that you follow the [Microverse README template](https://github.com/microverseinc/readme-template) and include all necessary sections.

## 🔴🚫Attention🚫🔴 

📢 Your Code Review Request will be marked as invalid🚫 in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadat3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_**📢💡Invalid Code Review Request **does not count** into the code reviews limit.💡**_

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    t.string 'name'
+    t.string 'photo'
+    t.string 'bio'
+    t.integer 'posts_counter'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+  end
+
+  add_foreign_key 'posts', 'users'
+end
```
Comment (omar-muhamad) [inline db/schema.rb:60]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md#:~:text=your%20project%20schema.-,Create%20and%20run%20the%20necessary%20migration%20files.,-Table%20and%20column), This `schema.rb` file should be generated using migration files which you didn't include in your app here. Please push the migration files that are used to generate this schema file regarding the following:
  - Please pay attention that in `posts `table the column holding the foreign key to the `users `table should be named `author_id`.
  - All columns that are foreign keys should have a corresponding index.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @mohamedgamaleldin999999,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by danmainah — APPROVED** [review]
Hi @mohamedgamaleldin999999 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Implemented the change requested to add migration files and ensure the user is referenced in the posts table using author_id

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**